src/test/datascience/notebook/helper.ts

```diff
@@ -410,6 +410,8 @@ export async function startJupyterServer(api?: IExtensionTestApi) {
         const uriString = decodeURIComponent(uri.toString());
         traceInfo(`Jupyter started and listening at ${uriString}`);
         await selector.setJupyterURIToRemote(uriString);
+        const notebookControllerManager = serviceContainer.get<INotebookControllerManager>(INotebookControllerManager);
+        (notebookControllerManager as any).allowRemoteConnection.resolve();
```

Comment:
```suggestion
        const notebookControllerManager = serviceContainer.get<NotebookControllerManager>(INotebookControllerManager);
        notebookControllerManager.allowRemoteConnection.resolve();
```

Reply:
Ahh, good call. Was trying to keep it out of the interface since it was just for tests. Forgot that I could just get the class here. I'll change that.